monkey/infection_monkey/monkey.py

```diff
@@ -86,12 +86,17 @@ def initialize(self):
         self._network = NetworkScanner()
         self._dropper_path = sys.argv[0]
 
+        server_num=0     
+        server_count = len(self._default_server)
```

Comment:
Doesn't make sense from readability perspective. If `_default_server` is in fact multiple servers we should rename it to `_default_servers`